docs/installation/enabling-agent-service.rst

```diff
@@ -6,25 +6,24 @@ Enabling the Flocker Agent Service
 
 .. begin-body-enable-agent-intro
 
-The Flocker agents, the ``flocker-dataset-agent`` and the ``flocker-container-agent``, are the workhorses of Flocker; you have them on each node in your cluster, and enabling them is an essential step in setting up your cluster.
+The ``flocker-dataset-agent`` is the workhorse of Flocker; you install an agent on each node in your cluster, and enabling them is an essential step in setting up your cluster.
+
+.. note:: The ``flocker-container-agent`` is now deprecated, but it can still be enabled.
+   For more information, see :ref:`deprecated-endpoints`.
 
 .. end-body-enable-agent-intro
 
 .. begin-body-enable-agent-main
 
-.. note::
-   Flocker's container management features depend on Docker.
-   You will need to make sure `Docker (at least 1.8) is installed`_ and running before you enable ``flocker-container-agent``.
-
-.. _Docker (at least 1.8) is installed: https://docs.docker.com/installation/
-
 CentOS 7, RHEL 7.2
 ==================
 
 #. Run the following commands to enable the agent service:
 
-   .. task:: enable_flocker_agent centos-7
-      :prompt: [root@agent-node]#
+   .. prompt:: bash [root@agent-node]#
+   
+      systemctl enable flocker-dataset-agent
+      systemctl start flocker-dataset-agent
```

Comment:
The `task` based console snippets are important. They are the link between the docs and our tests.

So instead we should modify:
 * https://github.com/ClusterHQ/flocker/blob/2a46750f9e990a6b57a71a9f237c25170fe6a256/flocker/provision/_install.py#L1117

Remove the flocker-container-agent lines.

This will likely break the acceptance tests, but we can then create a sub-issue to adjust the acceptance tests.